pdns/dnsdist-lua.cc

```diff
@@ -631,7 +631,8 @@ void setupLuaConfig(bool client)
 	SBind(sock, local);
 	SListen(sock, 5);
 	auto launch=[sock, local, password, apiKey, customHeaders]() {
-	  thread t(dnsdistWebserverThread, sock, local, password, apiKey ? *apiKey : "", customHeaders);
+          setWebserverConfig(password, apiKey, customHeaders);
+          thread t(dnsdistWebserverThread, sock, local);
 	  t.detach();
```

Comment:
Just a nit but indentation seems off.

Reply:
The file has been partially indented with tabs :/